SAMPLE_APPS.md

```diff
@@ -0,0 +1,81 @@
+# Try Out the Sample Apps
+
+Follow this guide to try out the sample app curated by Asgardeo. If you want to integrate Asgardeo for your own app, follow [this guide](/README.md#getting-started).
+
+
+## Table of Content
+
+-   [Create an Application in Asgardeo](#create-an-application-in-asgardeo)
+-   [Running the sample apps](#running-the-sample-apps)
+-   [Change the Application's Development Server Port](#change-the-applications-development-server-port)
+-   [Other Links](#other-links)
+
+
+## Create an Application in Asgardeo
+
+Before trying out the sample apps, you need to create an application in **Asgardeo**.
+
+1. Navigate to [**Asgardeo Console**](https://console.asgardeo.io/login) and click on **Applications** under **Develop** tab
+
+2. Click on **New Application** and then **Single Page Application**.
+
+3. Enter any name you prefer as the name of the app
+
+4. Add the redirect URL(s). The redirect URL usually is the URL in which your React application is hosted. For local environments, it will be something like `https://localhost:3000`. This will be used as a configuration in [Running the sample apps](#2-running-the-sample-apps) section.<img width="1439" alt="Screenshot 2022-09-14 at 17 26 57" src="https://user-images.githubusercontent.com/42619922/190148189-bb933d6b-2f8e-41e7-8c42-9d67e6746d17.png">
+
+4. Click on Register. You will be navigated to management page of the sample application. In the **Quick Start** tab, select **React** as the preferred technology.
+
+5. Since we are using sample app, select **Try out a sample** option and follow the guide.
+
+## Running the sample apps
+
+1. In the Quick Start guide, download the sample React application from the given link in the Asgardeo Console.
+
+2. Update configuration file `src/config.json` with your registered app details.
+
+```json
```

Comment:
Does this render properly without breaking the numbering?

Reply:
Fixed with 40a33d64ab9c2b43d86dbfa08f29cf827198b572